lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -247,6 +247,11 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
             this.currentPanel.expanded = true;
         }
 
+        if (changes.displayDefaultProperties?.currentValue) {
+            this.currentPanel.panelTitle = this.DefaultPanels.PROPERTIES;
```

Comment:
I think you can use `this.expandPanel(this.DefaultPanels.PROPERTIES)` here to avoid duplicating the code